support/spring-batch/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+dependencies {
+	compileOnly(libs.spring.boot3.batch)
```

Comment:
I recommend using the boot2 to check the minimum supported version of Spring Batch. 

And it's better to use the Spring Batch dependency directly than to use Spring Boot Batch. As you can see, there are no Spring Boot dependencies other than Test. The reason is that Spring Boot has too many dependencies and we have no control over them.

Reply:
Then i will create javax module using Spring Boot2 to check minimum supported version of Spring Batch. (Like spring-data-jpa module)

I changed to use Spring Batch dependency directly rather than using Spring Boot Batch dependency.